source/vmware/vmware-overview.rst

```diff
@@ -0,0 +1,380 @@
+
+vSAN Direct Drives
+   Allows creating independent datastores on every disk attached to a node,
+   where applications running on that node can have direct access to those
+   disks. The MinIO Tenant can fully utilize the available throughput of the
+   drive with none of the overhead associated to distributed storage.
+
+   vSAN Direct drives are ideal for clusters dedicated to running the MinIO
+   Tenant.
+
+While MinIO can make efficient use of either data path, vSAN Direct most closely
+mirrors MinIO best practices around ensuring the shortest possible path between
+a MinIO process and its connected storage.
+
+MinIO recommends using only *one* type of vSAN data path for all drives intended
+for use by a given MinIO Tenant. vSphere administrators should create
+a :vmware7-docs:`Storage Policy
+<com.vmware.vsphere.storage.doc/GUID-720298C6-ED3A-4E80-87E8-076FFF02655A.html>`
+associated to either the vSAN SNA or vSAN Direct disks and ensure that the
+:vmware7-docs:`namespace
+<vmware-vsphere-with-tanzu/GUID-1544C9FE-0B23-434E-B823-C59EFC2F7309.html>` in
+which the MinIO Tenant will be deployed has access to that policy.
+
+.. ToDo: Link to vmware docs on configuring/provisioning Direct/SNA Drives
+
+.. _minio-vmware-enable:
+
+Enable MinIO for VMware vSphere
+-------------------------------
+
+VMware vSphere 7 Update 1 includes the MinIO Client Plugin. You have to
+explicitly enable the plugin from the :guilabel:`Client Plugins` view in
+vCenter. 
+
+From the :guilabel:`Menu`, select :guilabel:`Administration`, then select
+:guilabel:`Client Plugins`. 
+
+IMAGE
+
+Select the :guilabel:`MinIO` checkbox and click :guilabel:`Enable` to enable the
+plugin. 
+
+IMAGE
+
+Use the :ref:`minio-vmware-getting-started` procedure to deploy a MinIO Tenant
+for evaluation and development. See :ref:`minio-vmware-create-tenant` for a
+complete guide to deploying MinIO Tenants using vCenter.
+
+.. _minio-vmware-getting-started:
+
+Getting Started
+---------------
+
+The following procedure creates a MinIO Tenant through the VMware vCenter
+interface. This procedure does not cover advanced configuration such as
+Server Side Encryption (SSE-S3) or external IDentity Providers (IDP). 
+See <Future Doc> for a complete description of all Tenant creation options.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+This procedure assumes the following:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+   :width: 100%
+
+   * - Component
+     - 
+
+   * - VMware vSphere v7 Update 1
+     - 
+
+   * - VMware ESXi Hosts
+     - Ensure the cluster has *at least* as many ESXi hosts as MinIO nodes
+       which you want to deploy.
+
+   * - :vmware7-docs:`Storage Policy 
+       <com.vmware.vsphere.storage.doc/GUID-720298C6-ED3A-4E80-87E8-076FFF02655A.html>` 
+     - MinIO ensures any PVCs generated to support the Tenant bind *only* to
+       PVs associated to the specified storage policy. 
+
+       The Storage Policy should point to a group of *either*:
```

Comment:
Storage policy is just a policy, this can reworded as Select either vSAN Direct or vSAN SNA based storage policy.